tests/test_fields.py

```diff
@@ -40,7 +40,7 @@ def test_include_resource_linkage_single(self, post):
         field = Relationship(
             related_url='/posts/{post_id}/author/',
             related_url_kwargs={'post_id': '<id>'},
-            include_resource_linkage=True, type_='people'
+            include_resource_linkage=True, type_='people', schema='PostSchema'
```

Comment:
This indicates a breaking change. Before this PR, `schema` was only a necessary argument if passing `include_data=True`. Could you add a check to see if a schema is available before trying to use its `get_attribute` method?